### PR TITLE
Add body to output parameters

### PR DESCRIPTION
**Pull request by lxwbr**
I'd like to use the body generated in the release for further usage in GHA steps. Not sure if this is the right way to do it.

# PR Notes
- [ ] Reviewed Checks: Verified output of Integration Tests
- [ ] Have set labels for release level

### CODE REVIEW

File: src/release_on_push_action/core.clj [hunk 1]
```diff
@@ -132,6 +132,7 @@
   [release-data]
   (printf "::set-output name=tag_name::%s\n" (:tag_name release-data))
   (printf "::set-output name=version::%s\n" (:name release-data)))
```
Comment (lxwbr) [inline src/release_on_push_action/core.clj:134]:
```suggestion
  (printf "::set-output name=version::%s\n" (:name release-data))
```

File: src/release_on_push_action/core.clj [hunk 2]
```diff
@@ -131,7 +131,8 @@
   "
   [release-data]
   (printf "::set-output name=tag_name::%s\n" (:tag_name release-data))
-  (printf "::set-output name=version::%s\n" (:name release-data)))
+  (printf "::set-output name=version::%s\n" (:name release-data))
+  (printf "::set-output name=body::%s\n" (:body release-data)))
```
Comment (lxwbr) [inline src/release_on_push_action/core.clj:135]:
```suggestion
  (printf "::set-output name=version::%s\n" (:name release-data))
  (printf "::set-output name=body::%s\n" (clojure.string/replace (:body release-data) #"\n" "%0A")))
```

**Comment by rymndhng** [comment]
Hey @lxwbr, sorry for the delay and thank you for the contribution 🎉 

I've taken your PR as a starting point and added a few small tweaks in #49. It's released in `v0.20.0` now.

**Comment by lxwbr** [comment]
@rymndhng thanks! I also see that you had to solve the multi-line output issue. I did something similar in a fork, but your solution is more elegant.

**Pull request closed without merging**